windows_containers/index.adoc

```diff
@@ -0,0 +1,33 @@
+[id="windows-container-overview"]
+= Windows container overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for {product-title} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. By using Red Hat subscription, you can get support for Windows Container. For more information see, xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[Release notes].
+
+xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Windows container workloads]:
```

Comment:
Optional suggestion: I would integrate this link into the description below instead of stating it by itself with a colon. Same for the other instances.

Reply:
I have integrated by stating - `For more information see...` and removed the links before the colon

Reply:
Please correct me if the wordings don't seem to be accurate

Reply:
Removed the colons and the links throughout the doc